federation/src/graphql/api/1_1/schema.ts

```diff
@@ -0,0 +1,9 @@
+import { NonEmptyArray } from 'type-graphql'
+import { AuthenticationResolver } from '../1_0/resolver/AuthenticationResolver'
+import { PublicCommunityInfoResolver } from '../1_0/resolver/PublicCommunityInfoResolver'
+import { SendCoinsResolver } from '../1_0/resolver/SendCoinsResolver'
```

Comment:
are relative imports allowed now?
eslint often not allows them

Reply:
In backend eslint was configured to disallow them, but not in federation. In Federation on multiple places relative imports are used.